spec/controllers/spree/admin/reports_controller_spec.rb

```diff
@@ -199,74 +211,104 @@
     before { login_as_admin }
 
     it "should build distributors for the current user" do
+      [c1, c2, s1, d1, d2, d3]
```

Comment:
I would rather create these c1, c2, etc within this "Products & Inventory" context. And if we could provide more meaningful names... it's really hard to understand what this vars are about :pray:

Reply:
> I would rather create these c1, c2, etc within this "Products & Inventory" context.

At the moment, there are several tests in this context that don't need these objects at all. I can create more contexts. Do you think that would improve readability?

> if we could provide more meaningful names...

Oh yes, totally agree. That would be great. I put that on my list.